README.md

```diff
@@ -33,6 +33,7 @@ Available frontends to define the folded circuit:
 
 - [arkworks](https://github.com/arkworks-rs), arkworks contributors
 - [Circom](https://github.com/iden3/circom), iden3, 0Kims Association
+- [Noname](https://github.com/zksecurity/noname/tree/main), a high-level programming language to write zkapps for kimchi or SnarkJS.
```

Comment:
proposal to follow the style of the previous links, putting the authors of the repo instead of a custom description
```suggestion
- [Noname](https://github.com/zksecurity/noname/tree/main), zkSecurity
```